backend/src/main/java/pl/krkteam/battleships/ships/placing/validation/services/ShipDirectionChecker.java

```diff
@@ -6,7 +6,7 @@
 class ShipDirectionChecker {
     static Direction whatDirection(ShipFromJson shipFromJson) {
         CoordinatesFromJson[] coordinatesFromJsonArray = shipFromJson.getCoordinates();
-        if (coordinatesFromJsonArray.length==1){
+        if (coordinatesFromJsonArray.length == 1) {
             return Direction.ONE_MAST;
         }
         if (coordinatesFromJsonArray[0].getX() + 1 == coordinatesFromJsonArray[1].getX() &&
```

Comment:
extract if expression to descriptive method

Reply:
dosen't make any sense